anitya/db/models.py

```diff
@@ -438,6 +441,30 @@ def __json__(self, detailed=False):
 
         return output
 
+    def set_check_times(self, session, last_check, next_check=None):
```

Comment:
I know the existing models have a lot of these kinds of functions, but it's an overly complicated way to do this and they should all be removed eventually. You can just set the values on the object directly.